src/live.rs

```diff
@@ -695,6 +696,37 @@ fn collect_iso_records(
     Ok(())
 }
 
+pub fn iso_extract_pxe(config: &IsoExtractPxeConfig) -> Result<()> {
+    let mut iso = IsoFs::new(&config.input)?;
+    let pxeboot = match iso.get_path(COREOS_ISO_PXEBOOT_DIR)? {
+        iso9660::DirectoryRecord::Directory(d) => d,
+        iso9660::DirectoryRecord::File(_) => {
+            bail!("expected dir {}, but found file", COREOS_ISO_PXEBOOT_DIR)
+        }
+    };
+
+    let outdir = config.output_dir.as_deref().unwrap_or(".");
```

Comment:
For clarity of help text, let's set a `default = "."` structopt property and avoid the unwrap here.